packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -306,9 +306,11 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
         ...updatableFieldInput,
         defaultValue:
           // Todo: we handle default value for all field types.
-          ![FieldMetadataType.SELECT, FieldMetadataType.BOOLEAN].includes(
-            existingFieldMetadata.type,
-          )
+          ![
+            FieldMetadataType.SELECT,
+            FieldMetadataType.MULTI_SELECT,
+            FieldMetadataType.BOOLEAN,
+          ].includes(existingFieldMetadata.type)
```

Comment:
I think we have an utility type that is checking is type is an enum, it's `isEnumFieldMetadataType`, but it includes `RATING` we must check if it's the correct behaviour

Reply:
As it is currently implemented, we cannot define a default value for rating. Think we can leave as it is now and create a dedicated utils when all fields are able to define a default value in the front